flood_adapt/dbs_controller.py

```diff
@@ -90,25 +90,26 @@ def get_aggregation_areas(self) -> dict:
 
         return aggregation_areas
 
-    def get_svi_map(self) -> gpd.GeoDataFrame:
-        """Get the geospatial social vulnerability index (SVI) data.
+    def get_static_map(self, path: Union[str, Path]) -> gpd.GeoDataFrame:
+        """Get a map from the static folder
+
+        Parameters
+        ----------
+        path : Union[str, Path]
+            Path to the map relative to the static folder
 
         Returns
         -------
         gpd.GeoDataFrame
-            SVI per aggregation area
+            GeoDataFrame with the map in crs 4326
         """
-        if self.site.attrs.fiat.svi:
-            svi_map = gpd.read_file(
-                self.input_path.parent
-                / "static"
-                / "site"
-                / self.site.attrs.fiat.svi.geom,
-                engine="pyogrio",
-            ).to_crs(4326)
-        else:
-            svi_map = None
-        return svi_map
+        # Read the map
+        full_path = self.static_path / path
+        if full_path.is_file():
+            return gpd.read_file(full_path, engine="pyogrio").to_crs(4326)
+
+        # If the file does not exist, return an empty GeoDataFrame
```

Comment:
if full_path.is_file(): will never throw an exception, it will just return true or false

So, shouldn't we raise an error or at least a warning? the exception can then be caught in get_svi_map()

Reply:
True, now throwing an error. Next step would be to propper exception handling in the API